doc/start/quick-cephfs.rst

```diff
@@ -164,9 +164,10 @@ by the mount.ceph helper program::
     sudo mount -t ceph {ip-address-of-MON}:{port-number-of-MON}:{path-to-be-mounted} -o name={user-name},secret={secret-key} {mount-point}
 
 If you have multiple file systems on your cluster you would need to pass
-``mds_namespace={fs-name}`` option to ``-o`` option to the ``mount`` command::
+``fs={fs-name}`` option (Or the deprecated ``mds_namespace=<fs-name>`` for
```

Comment:
Again, I'd leave out the part about mds_namespace here.

Reply:
Will fix it.